specification/frontdoor/resource-manager/Microsoft.Network/preview/2018-08-01-preview/frontdoor.json

```diff
@@ -1808,6 +1826,83 @@
             "name": "FrontDoorEnabledState",
             "modelAsString": true
           }
+        },
+        "redirectConfiguration": {
+          "description": "A reference to the redirect routing configuration. It is null for a forward-routing rule. But it must not be null if the routeType property is set to Redirect.",
+          "$ref": "#/definitions/RedirectConfiguration"
+        }
+      }
+    },
+    "RedirectConfiguration": {
+      "description": "The configuration for a redirect routing rule. This object is needed only if the type property of RoutingRule is set to Redirect. ",
+      "type": "object",
+      "properties": {
+        "redirectType": {
+          "description": "The redirect type the rule will use when redirecting traffic.",
+          "type": "string",
+          "enum": [
+            "Moved",
+            "Found",
+            "TemporaryRedirect",
+            "PermanentRedirect"
+          ],
+          "x-ms-enum": {
+            "name": "FrontDoorRedirectProtocol",
+            "modelAsString": true
+          }
+        },
+        "destinationProtocol": {
```

Comment:
The SharedSchema use "destinationSchema", better to be consistent
the other fields are already consistent